app/controllers/policies_controller.rb

```diff
@@ -34,28 +34,25 @@ def post
   protected
 
   def load_policy perform_automatic_deletion:, delete_permitted:, update_permitted:
-    policy_text = request.raw_post
 
-    policy_version = PolicyVersion.new \
-      role: current_user, policy: resource, policy_text: policy_text
-    policy_version.perform_automatic_deletion = perform_automatic_deletion
-    policy_version.delete_permitted = delete_permitted
-    policy_version.update_permitted = update_permitted
-    policy_version.save
-    loader = Loader::Orchestrate.new policy_version
-    loader.load
+    policy_ver = policy_version(
```

Comment:
In the method signature, remove poetry mode and add params.  

We have a lot of legacy poetry mode but I think any time we touch something for maintenance we need to fix.